packages/api/lib/rulesHelpers.js

```diff
@@ -141,15 +143,15 @@ function lookupCollectionInEvent(eventObject) {
 /**
  * Build payload from rule for lambda invocation
  *
- * @param {RuleRecord} rule           - API rule
+ * @param {PayloadRuleRecord} rule - API rule with possible cumulus_metadata object
  *
- * @returns {Promise<unknown>} lambda invocation payload
+ * @returns lambda invocation payload
  */
 async function buildPayload(rule) {
   // makes sure the workflow exists
   const bucket = getRequiredEnvVar('system_bucket');
   const stack = getRequiredEnvVar('stackName');
-  const workflowFileKey = workflows.getWorkflowFileKey(stack, rule.workflow);
+  const workflowFileKey = workflows.getWorkflowFileKey(stack, rule.workflow || '');
```

Comment:
Does this fix a specific issue where rule.workflow was coming in as undefined or something?

Reply:
No, it fixes the type assertions - rule.workflow can be undefined. 

That said, it should probably be more properly fixed as such: 

c5ac10b44f1